public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
Please change those values to prevent the browsers from caching the wrong values